jsmiddleware/src/main/java/com/segment/jsmiddleware/EdgeFunctionMiddleware.kt

```
@@ -77,9 +105,14 @@ class EdgeFunctionMiddleware(context: Context, localFile: String?) : JSMiddlewar
             runtime = JSRuntime(bundleStream)
             Log.d("JSMiddleware", bundleStream.toString())
 
+            if (runtime?.getObject("edge_function") == null) {
```

fail early here. since this is a requirement